libraries/engage/cart/components/PaymentBar/PaymentBarReserveButton.jsx

```diff
@@ -22,11 +23,14 @@ function PaymentBarReserveButton({ historyReset }: Props) {
   const { flags: { orderable } } = React.useContext(CartContext);
 
   const handleClick = React.useCallback(() => {
-    FulfillmentSheet.open((location, orderSuccess) => {
-      if (orderSuccess === true) {
-        historyReset();
-      }
-    }, 1);
+    FulfillmentSheet.open({
+      stage: STAGE_RESERVE_FORM,
+      callback: (location, productId, orderSuccess) => {
```

Comment:
Strange to have productId here. Which is actually empty for cart reservation

Reply:
well, it is needed in all other places and can be `null`.